app/controllers/admin_shelters_controller.rb

```diff
@@ -0,0 +1,30 @@
+class AdminSheltersController < ApplicationController
```

Comment:
You would want to create a directory called `admin` and then within that directory have a `shelters` controller and an `applications` controller. 
```ruby 
app/controllers/admin/shelters_controller.rb
app/controllers/admin/applications_controller.rb
``` 
Then your class would read
```ruby
class Admin::SheltersController < ApplicationController
  def index
     @shelters = Shelter.list_alphabetically
     @pending_applications = Shelter.pending_applications
  end
end
```
in here is where you will call your model method to alphabetize all shelters. As well as possibly call a method that you create to get all pending applications